common/src/Model/PortalFile.cpp

```diff
@@ -89,14 +103,19 @@ void PortalFile::load(const IO::Path& path) {
   // read portals
   for (int i = 0; i < numPortals; ++i) {
     std::getline(stream, line);
-    const auto components = kdl::str_split(line, "() \n\t\r");
+    const auto components = kdl::str_split(line, lineSplitter);
 
     if (!stream.good() || components.size() < 3) {
       throw FileFormatException("Error reading portal");
     }
 
     std::vector<vm::vec3f> verts;
-    size_t ptr = 3;
+    size_t ptr;
```

Comment:
Please change this to

```
const auto ptr = prt1ForQ3 ? 4u : 3u; 
```

Reply:
I'll leave this non-const BTW since it does get incremented later. I can leave it `size_t` as it was, or switch it to `auto` if you like. I.e.
```c++
size_t ptr = prt1ForQ3 ? 4u : 3u;
```
or
```c++
auto ptr = prt1ForQ3 ? 4u : 3u;
```

Reply:
(Eh, I'll assume auto for now since you explicitly mentioned that above about auto variables. Can change back if you prefer.)